### ISSUE TITLE
Tide-search "peptide-centric-search" output is missing "xcorr score" values.

### ISSUE THREAD

**Issue by wsnoble** [crux-toolkit/crux-toolkit#575]
Reported on crux-users:

When I choose a peptide-centric-search, the "tide-search.target.txt" has a header line that includes "xcorr score" but that field is completely missing from the following lines.  That makes interpreting the columns hard, but more important, it prevents me from screening by xcorr score.  I tried many options like "score-function=both" or "score-function=residue-evidence" that always lead to a dead end of either a "not implemented for peptide-centric-search" or a complaint about exact-p-value requiring a specific mz-bin-width setting, or else a setting to use-neutral-loss-peaks=false, but nothing I did would restore the "xcorr score" column (only its header line).    Suggestions appreciated.  I am running Crux version 4.1-6c8889b7-2022-05-23

**Comment by kfattila** [comment]
Hi, This would be an easy-peasy fix, but I need some instruction, Bill.

The XCorrs are actually there. The problem is in the headers. There are no delta_lcn scores reported. The mistake is that, this should not be written to the headers.

I implemented this peptide-centric-search when I was in your lab, and I have no idea why we did not report the delta_lcn score. maybe there was a reason for that.

We can do:
1) remove the "delta_lcn" from the headers and the problem is fixed, OR
2) calculate the delta_lcn and report them.

How shall I proceed? 1 or 2?
